src/models/pocketSave.ts

```diff
@@ -58,6 +49,23 @@ export class PocketSaveModel {
     return pocketSave;
   }
 
+  /**
```

Comment:
this covers this ticket? https://getpocket.atlassian.net/browse/INFRA-810

Reply:
Yes, there is some overlap here 😳 . I imagine the resolver would call this method, but the schema still needs to be updated and integration tests for that specific resolver added.

Reply:
Oh and the schema has to be checked -- as per my comment I think that resolver might return a union type with NotFound.